### ISSUE TITLE
Start RabbitMQ service optionally in windows-exe installer

### ISSUE THREAD

**Issue by inikulshin** [rabbitmq/rabbitmq-server#2702]
It's a very common practice - to ask user whether to run/launch installed software or not.

Sometimes, after upgrade service don't have a chance to be started immediately after installing - for example, if some enabled plugins are custom and must be copied to RABBITMQ_HOME before starting.

We (Varonis) are also interested in disabling `packaging/windows-exe/rabbitmq_nsi.in`'s `rabbitmq-service.bat start` it in **silent mode** - either conditionally, with command line flag, or unconditionally (`IfSilent`).

**Comment by michaelklishin** [comment]
This is similar to the approach used by some Linux distributions, e.g. Debian. Would you be interested in contributing some changes to the installer since you both understand the problem quite well and run on Windows in production? (FWIW our team are hardly Windows experts)

**Comment by lukebakken** [comment]
Please add a command line flag to _disable_ auto-start of the Windows service. Keep the default behavior of auto-starting. Thank you.

**Comment by inikulshin** [comment]
@lukebakken @michaelklishin

Hi,
I'm interested to contribute, but I can't find those files in `rabbitmq-server` repository. They were in archived `rabbitmq-server-release`, but where are they now?

Will it be possible to update **3.8.x** also?

**Comment by lukebakken** [comment]
Yep, I figured I'd move them once a PR was submitted but I can do it now.
